GoogleUtilities/AppDelegateSwizzler/GULAppDelegateSwizzler.m

```diff
@@ -112,6 +92,35 @@ typedef void (*GULRealDidReceiveRemoteNotificationWithCompletionIMP)(
 /** The original instance of App Delegate. */
 static id<UIApplicationDelegate> gOriginalAppDelegate;
 
+/** The original App Delegate class */
+static Class gOriginalAppDelegateClass;
+
+/** The subclass of the original App Delegate. */
+static Class gAppDelegateSubclass;
+
+/** Remote notification methods selectors
+ *
+ *  We have to opt out of referencing APNS related App Delegate methods directly. If we do, then
```

Comment:
We have to opt out of referencing APNS related App Delegate methods directly to prevent an Apple review warning email about missing Push Notification Entitlement (like here ...